dist/static/site.js

```diff
@@ -0,0 +1,186 @@
+/** The name of the user */
+let name = null;
+
+/** The WebSocket connection */
+let sock = null;
+
+/** Heartbeat variable */
+let isAlive = true;
+
```

Comment:
I'm hesitant to add this, since the only user input that could possibly cause an XSS attack is the name of the user, which we already sanitize. Sure the song title could contain one, but it's not like youtube would allow a song with a title that is a script, which means that this would only happen if we couldn't trust the server.